app/components/AlbumArt/index.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import React from 'react';
```

Comment:
lets not add newlines after every import

Reply:
got it!